### ISSUE TITLE
Pull Resquest -  Validate JSONs / verify-json-validation (pull_request) 

### ISSUE THREAD

**Issue by KosmosKarl** [Cardano-NFTs/policyIDs#2750]
I have submitted a pull request.  
The pull request has a cross next to it caused by  'Validate JSONs / verify-json-validation (pull_request) '
I am not a developer and dont understand the issue here.  Can someone explain if this is an issue and if so how i can fix it.

All checks have failed
1 failing check
@github-actions
Validate JSONs / verify-json-validation (pull_request) Failing after 11s — verify-json-validation

**Comment by ThianchaiNFT** [comment]
same here.

**Comment by sharnold52** [comment]
Oh same here as well.

**Comment by ThianchaiNFT** [comment]
Write it in this format (exactly symbols, line and space, just change your project name and policies)
I did correct this and re submitted. All passed. Hope this will help you.
https://support.cnft.io/article/verify-your-project

This is my PR#2785, I have to tried several times. Now All checks have passed.
#2785

**Comment by sharnold52** [comment]
Yep, the check seems to be working correctly now. I didn't change any of my files but it is passing now as well.

**Comment by ThianchaiNFT** [comment]
> Yep, the check seems to be working correctly now. I didn't change any of my files but it is passing now as well.

Great.;-)
